modules/product/product.resolver.js

```diff
@@ -22,7 +22,18 @@ const productsQuery = {
       };
     }
   },
-  getModelsbyCategory: (parent, args) => productsService.getModelsByCategory(args.id)
+  getModelsByCategory: async (parent, args) => {
+      const models = await productsService.getModelsByCategory(args.id)
+      if (models) {
+        return models;
+      }
+      return {
```

Comment:
It should not return an error if no models are found, we just need to return empty array